.github/workflows/deploy.yml

```diff
@@ -74,6 +74,8 @@ jobs:
         cp -v tmp_artifacts/tobira .deployment/files/
         cp -v tmp_artifacts/util/dev-config/logo-large.svg .deployment/files/
         cp -v tmp_artifacts/util/dev-config/logo-small.svg .deployment/files/
+        cp -v tmp_artifacts/util/dev-config/logo-large-dark.svg .deployment/files/
+        cp -v tmp_artifacts/util/dev-config/logo-small-dark.svg .deployment/files/
```

Comment:
Well, we don't have `small-dark` for our test deployment, so only `large-dark` needs to be copied.